src/pairwiseFunctors/FlopCounterFunctor.h

```diff
@@ -165,12 +183,10 @@ class FlopCounterFunctor : public Functor<Particle, ParticleCell> {
   unsigned long getKernelCalls() const { return _kernelCalls; }
 
  private:
-  enum SoAAttributes { id, posX, posY, posZ };
-
   double _cutoffSquare;
   unsigned long _distanceCalculations, _kernelCalls;
+  // 3 sub + 3 square + 2 add
+  static constexpr double numFlopsPerDistanceCalculation = 8.0;
 };
 
-} /* namespace autopas */
-
-#endif /* SRC_PAIRWISEFUNCTORS_FLOPCOUNTERFUNCTOR_H_ */
+} /* namespace autopas */
```

Comment:
I would prefer
``` C++
}  // namspace autopas
```
in general. What is you opinion? I think that is also what google wants

Reply:
I actually have no preference here